controllers/uaa.go

```diff
@@ -38,6 +42,51 @@ func (c *UAAContext) InviteUsers(rw web.ResponseWriter, req *web.Request) {
 	c.uaaProxy(rw, req, reqURL, true)
 }
 
+// SendInvite sends users an email with a link to the UAA invite
+func (c *UAAContext) SendInvite(rw web.ResponseWriter, req *web.Request) {
```

Comment:
This function is pretty large and isn't unit tested. It might make sense to write some tests for specific functionality and start splitting up this function?